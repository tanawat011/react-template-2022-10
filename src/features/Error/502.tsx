export const Error502 = () => {
  return <div>502 Bed Gateway</div>
}
