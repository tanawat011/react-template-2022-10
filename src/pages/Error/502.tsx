import { allRoutes } from 'Routes'
import { Button } from 'components/Button'

export const Error502 = () => {
  return (
    <div>
      <p>502 Bed Gateway</p>

      <Button link to={`${allRoutes.backoffice.fullPath}`}>
        Back to Home Page
      </Button>
    </div>
  )
}
