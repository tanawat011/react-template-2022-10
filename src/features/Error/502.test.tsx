import { render } from '@testing-library/react'

import { Error502 } from './502'

describe('<Error502 />', () => {
  test('match snapshot', () => {
    const { asFragment } = render(<Error502 />)
    expect(asFragment()).toMatchSnapshot()
  })
})
