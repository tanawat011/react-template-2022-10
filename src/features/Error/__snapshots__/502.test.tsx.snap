// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error502 /> match snapshot 1`] = `
<DocumentFragment>
  <div>
    502 Bed Gateway
  </div>
</DocumentFragment>
`;
